GetandTake/Pages/Products/Edit.cshtml.cs

```diff
@@ -0,0 +1,61 @@
+using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+using GetandTake.Services.Abstract;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace GetandTake.Pages.Products;
+
+public class EditModel : PageModel
+{
+
+    private readonly IProductService _productService;
```

Comment:
There is also a good code practice for private fields and properties:
1. **Fields** should **not be separated** by empty rows;
2. **Properties** should **be separated** by empty rows (because modifiers may include additional logic it would make code more difficult to read);

In our case, it would look like this:
```suggestion
    private readonly IProductService _productService;
    private readonly ICategoryService _categoryService;
    private readonly ISupplierService _supplierService;
    
    public ProductsDTO ProductDTO { get; set; }
    
    [BindProperty]
    public ProductDTO Product { get; set; }
    
    public IEnumerable<Category> Categories { get; private set; }
    
    ...
```